content/changelog/2024-10-17-sozu-1.0.5.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sōzu 1.0.5 is available and deployed
+date: 2024-10-17
+tags:
+  - sozu
+authors:
+  - name: David Legrand
+    link: https://github.com/davlgd
+    image: https://github.com/davlgd.png?size=40
+  - name: Florentin Dubois
+    link: https://github.com/FlorentinDUBOIS
+    image: https://github.com/FlorentinDUBOIS.png?size=40
+description: Lots of fixes and improvements before the 1.1.0 release
+excludeSearch: true
+---
+
+[Sōzu](https://www.sozu.io) 1.0.5 is now available and deployed on `cleverapps.io` domains, we're progressively rolling it out on our own domains and dedicated load balancers. Since [the release of Sōzu 1.0.2](/changelog/2024-10-04-sozu-1.0.2/) we've fixed some bugs, improved metrics, CLI and metrics.
```

Comment:
```suggestion
[Sōzu](https://www.sozu.io) 1.0.5 is now available and deployed on `cleverapps.io` domains, we're progressively rolling it out on our own domains and dedicated load balancers. Since [the release of Sōzu 1.0.2](/changelog/2024-10-04-sozu-1.0.2/) we've fixed some bugs, improved metrics and CLI.
```